src/js/components/ContentComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react/addons";
+
+import Util from "../helpers/Util";
+import SectionComponent from "../components/SectionComponent";
+
+var ContentComponent = React.createClass({
+  "displayName": "ContentComponent",
+
+  propTypes: {
+    active: React.PropTypes.string,
+    children: Util.getComponentTypeValidator(SectionComponent),
+    className: React.PropTypes.string
+  },
+
+  renderChildren: function () {
+    var {active, children} = this.props;
+
+    return React.Children.map(children, (child) =>
+      React.addons.cloneWithProps(child, {
+        active: child.props.id === active
```

Comment:
Oh I see. Can we use an actual property rather than the HTML `id`? It's not really idiomatic, imho.